amplify.yml

```diff
@@ -1,19 +0,0 @@
-version: 1
```

Comment:
Why on earth is this file being removed?

Reply:
This file needs to be restored so it doesn't break the aws amplify deploy

Reply:
- Restored amplify.yml file so it doesn't break the aws amplify deployment
- Grabbed the package.json from the main branch and re-installed the new modules.